root/etc/e-smith/events/actions/restore-config-reinstall

```diff
@@ -19,21 +19,16 @@
 # along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+packagelist_path=/var/lib/nethserver/backup/package-list
 
-if [ -f /var/lib/nethserver/backup/package-list ]; then
+if [ ! -f ${packagelist_path} ]; then
+    echo "[WARNING] restore-config-reinstall: package-list not found, skipped"
+    exit 0
+fi
 
-    # Migrate packages from NS 6 to NS 7
-    # See: http://docs.nethserver.org/en/v7/release_notes.html#discontinued-packages
-    sed -i \
-       -e 's/^nethserver-vpn$/nethserver-openvpn\nnethserver-ipsec-tunnels/'  \
-       -e 's/^nethserver-collectd-web$/nethserver-cgp/' \
-       -e 's/^nethserver-fetchmail$/nethserver-getmail/' \
-       -e 's/^nethserver-snort$/nethserver-suricata/' \
-       -e 's/^nethserver-ibays$/nethserver-virtualhosts\nnethserver-samba/' \
-       -e '/^nethserver-c-icap$/d' \
-       -e '/^nethserver-ipsec$/d' \
-    /var/lib/nethserver/backup/package-list
+# Migrate packages from NS 6 to NS 7
+# See: http://docs.nethserver.org/en/v7/release_notes.html#discontinued-packages
+sed -i -f /etc/nethserver/ns6upgrade-package-list.sed ${packagelist_path}
```

Comment:
If the `ns6upgrade-package-list.sed` file is not used somewhere else, I'd prefer to leave the code as is to improve readability.

Reply:
It is required by `restore-config-download`. I'd prefer not duplicating it!